CRM/Upgrade/Incremental/MessageTemplates.php

```diff
@@ -350,6 +350,14 @@ protected function getTemplateUpdates() {
           ['name' => 'contribution_offline_receipt', 'type' => 'html'],
         ],
       ],
+      [
+        'version' => '5.53.alpha1',
+        'upgrade_descriptor' => ts('Fix non-php compliant dates'),
+        'templates' => [
+          ['name' => 'event_online_receipt', 'type' => 'subject'],
```

Comment:
do we not need to do html version as well?

Reply:
opps copy & paste - do you want me to fix it or are you thinking we shouldn't do this?

Reply:
fix it